Makefile

```diff
@@ -0,0 +1,45 @@
+MAKE   = make
+PYTHON = python
+SETUP  = $(PYTHON) ./setup.py
+
+.PHONY: clean cleandocs coverage dist docs opendocs unit-coverage upload
+
+help:
+	@echo "Usage: \`make <target>' where <target> is one or more of"
+	@echo "  clean          delete intermediate work product and start fresh"
+	@echo "  cleandocs      delete cached HTML documentation and start fresh"
+	@echo "  coverage       report overall test coverage"
+	@echo "  docs           build HTML documentation using Sphinx (incremental)"
+	@echo "  opendocs       open local HTML documentation in browser"
+	@echo "  dist           generate source and wheel distribution into dist/"
+	@echo "  unit-coverage  report unit test coverage"
+	@echo "  upload         upload distribution to PyPI"
+
+clean:
+	find . -type f -name \*.pyc -exec rm {} \;
+	find . -type f -name .DS_Store -exec rm {} \;
+	rm -rf dist .coverage
+
+cleandocs:
+	$(MAKE) -C docs clean
+
+coverage:
+	pytest --cov-report term-missing --cov=src --cov=tests
+
+dist:
+	rm -rf dist/
+	$(SETUP) sdist bdist_wheel
+
+docs:
+	$(MAKE) -C docs html
```

Comment:
Maybe I am missing something, but where is Sphinx configured? I mean, is this enough?

Reply:
it's for future, so i have in mind that we wanna have sphinx and when we wanna release a new doc version we already have the make command to do that

Reply:
Yes we want to have sphinx for sure! But having the command here suggests that NOW it's possible to build docs. Maybe we can comment `$(MAKE) -C docs html` and have something like `@echo "Feature not implemented yet"`